package.json

```diff
@@ -80,7 +80,7 @@
     "@material/floating-label": "^1.1.0",
     "@material/icon-button": "^2.1.1",
     "@material/line-ripple": "^1.0.0",
-    "@material/list": "1.0.0",
+    "@material/list": "^1.0.0",
```

Comment:
@moog16 sub-packages include any `1.x` branch variant, and we have type errors without this change as code expects later versions

Reply:
Should the `@material/list` dep update in the `@material/react-list` package instead?

Reply:
It's already using the ^, only the one here wasn't